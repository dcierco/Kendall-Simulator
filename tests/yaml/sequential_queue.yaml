numbers:
  - 0.1234
  - 0.5678
  - 0.9012
  - 0.3456
  - 0.7890
  - 0.2345
  - 0.6789
  - 0.0123
  - 0.4567
  - 0.8901
  - 0.2345
  - 0.6789
  - 0.0123
  - 0.4567
  - 0.8901

queuesList:
  - name: Q1
    servers: 1
    capacity: 3
    arrivalTimeMin: 1.0
    arrivalTimeMax: 2.0
    serviceTimeMin: 2.0
    serviceTimeMax: 4.0
    arrivalStartTime: 0.0
    network:
      - [Q2, 1.0]

  - name: Q2
    servers: 2
    capacity: 4
    serviceTimeMin: 3.0
    serviceTimeMax: 5.0
    network:
      - [Q3, 1.0]

  - name: Q3
    servers: 1
    capacity: 2
    serviceTimeMin: 1.0
    serviceTimeMax: 3.0
