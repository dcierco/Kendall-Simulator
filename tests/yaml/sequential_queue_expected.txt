Queue: Q1 (G/G/1/3/∞/FCFS):
State: 1, Time: 2.7900000000000005, Probability: 31.5204%
State: 2, Time: 4.667799999999996, Probability: 52.7352%
State: 3, Time: 1.3936000000000028, Probability: 15.7444%
Number of remaining clients: 3
Losses: 0
Queue: Q2 (G/G/2/4/∞/FCFS):
State: 0, Time: 2.2468, Probability: 25.3836%
State: 1, Time: 2.829000000000001, Probability: 31.961%
State: 2, Time: 3.775599999999999, Probability: 42.6554%
Number of remaining clients: 1
Losses: 0
Queue: Q3 (G/G/1/2/∞/FCFS):
State: 0, Time: 7.3824, Probability: 83.4038%
State: 1, Time: 1.4690000000000003, Probability: 16.5962%
Number of remaining clients: 1
Losses: 0
Simulation time: 8.8514 seconds
Total losses: 0
